docker/env.sh

```diff
@@ -13,6 +13,7 @@ if [[ "$*" == *"--go-docker"* ]]; then
     export IN_DOCKER_GO=true
     echo "Using docker-internal infra."
 else
+    export OTLP_ENDPOINT=http://localhost:4318
```

Comment:
in the `compose.hobby.yml` deploy, i believe this needs to be `http://collector:4318` since the collector and the backend are both running in docker and do not have the same `localhost`

Reply:
Unless I'm mistaken hobby version should utilize .env which specifies `http://collection:4318` and the above is an override for local run. It only gets executed on the else statement of the if `--go-docker` check (It's provided when running `./run-hobby.sh` but not when running `./run-backend.sh`. 

Please let me know if I'm mistaken here and can modify.

Reply:
ah that's correct, my mistake! appreciate the attentiveness here